<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-SNILOAU" failures="0" tests="2" name="org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout" time="1.065" errors="1" timestamp="2024-08-27T09:51:13 BDT" skipped="0">
  <testcase classname="org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout" name="testMethod1" time="0.013"/>
  <testcase classname="org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout" name="testMethod2" time="1.052">
    <error message="Method org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout.testMethod2() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout.testMethod2() didn't finish within the time-out 1000
at java.base@17.0.9/java.lang.Thread.sleep(Native Method)
at app//org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout.testMethod2(TC010_TestNG_TestMethodDisabledTimeout.java:27)
at java.base@17.0.9/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.9/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.9/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.9/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.9/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.9/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.9/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.9/java.lang.Thread.run(Thread.java:840)
]]>
    </error>
  </testcase> <!-- testMethod2 -->
</testsuite> <!-- org.nadim.testscripts.TC010_TestNG_TestMethodDisabledTimeout -->
